polling_stations/apps/core/tests/test_slack_client.py

```diff
@@ -1,5 +1,6 @@
 from unittest.mock import Mock, patch
 import pytest
+from six import StringIO
```

Comment:
Wrong import here...we don't want to rely on `six`, so I think this is `io.StringIO` now